### DISCUSSION TITLE
Need help adding a request

### DISCUSSION THREAD

**Discussion by Eboreg2** [PygmalionOfCyprus/cmo-db-requests#4323] [Q&A] [answered]
There's a certain piece of equipment that I would like to add but I'm not entirely sure where to add it and what to put in for its specifications. The platform in question is the R-330Zh Zhitel, a Russian truck-mounted electronic warfare system designed to not only jam enemy communications but also GPS navigation systems. Apparently, it's been shown to be quite effective at protecting targets from attack by GPS-guided bombs which I'm sure would appeal to many scenario designers wanting to complicate things that little bit more. Could anyone walk me through this?

**Comment by claudejdev** [comment, marked as answer]
Hey there,

check the database to see if it's already there. Probably as a facility.
If not, try to find the closet platforms and open a new copy request.
Provide the sources regarding the performances as they are known.
Multiples sources as much as possible, certainly if you notice contradictory figures.

Your request will probably include a couple of generic jammers. Make sure to identify the era during which they were designed.

Have fun 😊
